src/qt/walletmodeltransaction.cpp

```diff
@@ -22,14 +22,14 @@ QList<SendCoinsRecipient> WalletModelTransaction::getRecipients() const
     return recipients;
 }
 
-std::unique_ptr<interfaces::PendingWalletTx>& WalletModelTransaction::getWtx()
+CTransactionRef& WalletModelTransaction::getWtx()
 {
     return wtx;
 }
 
 unsigned int WalletModelTransaction::getTransactionSize()
 {
-    return wtx ? ::GetSerializeSize(wtx->get(), SER_NETWORK, PROTOCOL_VERSION) : 0;
+    return wtx != nullptr ? ::GetSerializeSize(wtx.get(), SER_NETWORK, PROTOCOL_VERSION) : 0;
```

Comment:
adding the `!= nullptr` is okay, maybe even desired, but is also kinda unrelated here

Reply:
16415: also
```suggestion
    return wtx != nullptr ? ::GetSerializeSize(*wtx, SER_NETWORK, PROTOCOL_VERSION) : 0;
```

Reply:
AFAIK, the lack of `!= nullptr` causes a build failure and was therefore added